vcs/git/clone_update.go

```diff
@@ -45,7 +45,10 @@ func init() {
 }
 
 func Clone(url, dir string, opt vcs.CloneOpt) (vcs.Repository, error) {
-	clopt := git2go.CloneOptions{Bare: opt.Bare}
+	clopt := git2go.CloneOptions{
+		FetchOptions: &git2go.FetchOptions{},
```

Comment:
Gotcha. Just realized that too. In that case, would it be more clear to replace that line with this?

```Go
clopt.FetchOptions.RemoteCallbacks = *rc
```